testsuite/forge/src/backend/k8s/cluster_helper.rs

```diff
@@ -394,6 +395,12 @@ pub async fn uninstall_testnet_resources(kube_namespace: String) -> Result<()> {
     Ok(())
 }
 
+fn generate_new_era() -> String {
+    let mut rng = rand::thread_rng();
+    let r: u8 = rng.gen();
+    format!("forge{}", r)
```

Comment:
Doesn't the era need to be an integer?

Reply:
nope. under the hood, it's just used to name kubernetes resources, so it's any string